scripts/markdown/check-edit-links.js

```diff
@@ -0,0 +1,163 @@
+const fs = require('fs').promises;
+const path = require('path');
+const fetch = require('node-fetch-2');
+const editUrls = require('../../config/edit-page-config.json');
+const { pause } = require('../dashboard/build-dashboard');
+
+const ignoreFiles = [
+  'reference/specification/v2.x.md',
+  'reference/specification/v3.0.0-explorer.md',
+  'reference/specification/v3.0.0.md'
+];
+
+/**
+ * Process a batch of URLs to check for 404s
+ * @param {object[]} batch - Array of path objects to check
+ * @returns {Promise<string[]>} Array of URLs that returned 404
+ */
+async function processBatch(batch) {
+  const TIMEOUT_MS = 5000;
```

Comment:
Make it configurable with env variable as well.

Reply:
Are we over engineering things here? Why do we need env variable support for that?

Reply:
done